### PR TITLE
Feature/86 integration tests

### PR DESCRIPTION
**Pull request by gregdeane**
#86 

Notice that I switched `accessibleProducts` in navigation.html. I'm not entirely sure why I had to switch but it seems to work. Maybe scrutinize this part to ensure that the change is stable.

Also, temporary code had to be written to strip the value "string" from the url properties that are returned from the backend. The temp code is contained in `module-connector.service.js`. We should get BE to update the data so we can remove this temp code.

## Important
The integration tests represented by this PR are limited in scope and prove only that we're on the right track for navigating between one app and another. Much testing is still required and should naturally come as we move forward.

### CODE REVIEW

File: src/app/common/backend-connector/module-connector.service.js [hunk 1]
```diff
@@ -7,9 +7,29 @@ class ModuleConnectorService {
...
+  }
+
+  getTempAccessibleProducts() {
     let url = this.baseUrl + '/products';
 
-    return this.$http.get(url);
+    const updateUrls = (products) => {
+      products.forEach(product => {
+        product.applications.forEach(application => {
+          application.url = '';
```
Comment (frontendherodk) [inline src/app/common/backend-connector/module-connector.service.js]: I'm probably missing some context here, but why are we setting the application urls to an empty string here?

Comment (dami-gg) [inline src/app/common/backend-connector/module-connector.service.js]: everything is temporal for the integration tests only (plus a small set of fixes we should transfer when we're done)

**Comment by dami-gg** [comment]
This is temporarily blocked, we'll merge it later when we're done with the tests if needed

**Comment by dami-gg** [comment]
:-1:

**Comment by dami-gg** [comment]
Closing since we have addressed those topics in different PRs

**Pull request closed without merging**